### DISCUSSION TITLE
Seriously upgraded XT

### DISCUSSION THREAD

**Discussion by Randrianasulu** [jbruchon/elks#1559] [General]
https://sites.google.com/site/misterzeropage/

I also "just learned" there were 386 or even 486 upgrades for AT machines ...o.O

https://www.cpushack.com/2019/08/14/how-to-386-your-at-intel-inboard-386-at/

with pictures.

https://dosdays.co.uk/topics/cpu_upgrade_options.php
for 286->486 there is text only mentions...

**Comment by ghaerr** [comment]
Those look like pretty cool options for upgrading an XT. Let us know if you try booting ELKS on any of them, they should all work. :)

**Comment by Randrianasulu** [comment]
I think 86box by now have emulation of such machines (in its git version or fork)

https://www.reddit.com/r/vintagecomputing/comments/1vg70dt/windows_95_finally_running_on_an_intel_inboard/
